src/main/resources/application.yml

```
@@ -155,6 +156,8 @@ ide-sidecar:
     enabled: true
     # How often to check?
     interval-seconds: 60
+  proxy:
+    ccloud-api-regex: "(/artifact.*)|(/sql.*)|(/fcpm/v2/compute-pools.*)"
```

Isn't `/sql.*` a flink data plane API? We should rename this field to `ccloud-api-control-plane-regex` and remove `/sql.*` from it.
```suggestion
  proxy:
    ccloud-api-regex: "(/artifact.*)|(/fcpm/v2/compute-pools.*)"
```

---

good catch!